### ISSUE TITLE
Is this a bug that the logo is not completely visible in p5js.org website

### ISSUE THREAD

**Issue by pankaj-2503** [processing/p5.js#6700]
## Here is how is looks like:
![image](https://github.com/processing/p5.js/assets/90036864/b1d59861-7706-42be-81fd-98ad7f4831f0)

### Most appropriate sub-area of p5.js?

- [X] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [X] Data
- [ ] DOM
- [X] Events
- [X] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### Feature enhancement details

logo should be clearly visible  in  footer by manipulating it using css..

**Comment by paulpriyanshu** [comment]
Hi @pankaj-2503 
I think it is not a bug or it will be too obvious and someone would have fixed it still wait and see the response of the maintainers if they want it to be fixed.

**Comment by davepagurek** [comment]
We're actually in the process of redesigning the website! I'm going to close this for organizational purposes because all of this will look different soon. Stay tuned 👀
